utilities/src/main/java/ru/study21/jcsv/xxl/io/CSVWriter.java

```diff
@@ -0,0 +1,7 @@
+package ru.study21.jcsv.xxl.io;
+
+import java.util.List;
+
+public interface CSVWriter {
+    void writeRow(List<String> row);
+}
```

Comment:
CSVCustomizableAnalyzer was merged just recently and its CSVCustomizableAnalyzer.Action interface should used here
as center of implementation. I mean that you should implement class implementing CSVCustomizableAnalyzer.Action.

CSVCustomizableAnalyzer.Action in it's current form is not perfect to implement wrting offset because it is hardly possible to know it from list of rows passed to action. So as part of this issue you should also add new parameter - let's name it offset - to action method. (I don't want to pass CSVReader there - to be sure that no action will try to read next record). And may by we should introduce record type State to keep two fields: offset and rows that could be extended later without changing of interface signature.

CSVReader should be extended to return current offset

Another change of to CSVCustomizableAnalyzer.Action is needed: we need to catch moment when CSV records are over. So let's add method finish to interface and trigger it when it happens.

After all that what remains is to implement CSVCustomizableAnalyzer.Action passing it OutputStream via constructor, implementing action method to output offset (with some kind of buffering) and implementing finish method to flush buffers and close file.